manuscript/recipes/photoprism.md

```diff
@@ -0,0 +1,176 @@
+hero: Your own private google photos
+
+# Photoprism
+
+[Photoprism™](https://github.com/photoprism/photoprism) *" is a server-based application for browsing, organizing and sharing your personal photo collection. It makes use of the latest technologies to automatically tag and find pictures without getting in your way. Say goodbye to solutions that force you to upload your visual memories to the cloud."*
```

Comment:
The formatting here is making the entire paragraph appear italicized

Reply:
I put it italiced as it is direct quote translation / reference of the original photoprism text. In spanish quotes are itialicized, should I remove it then?

Reply:
Yes please - the convention I've followed in other recipes has been simply to quote (but not italicise) content I've taken from an upstream source, i.e. https://geek-cookbook.funkypenguin.co.nz/recipes/ghost/